apps/api/src/app/change/usecases/promote-notification-template-change/promote-notification-template-change.usecase.ts

```diff
@@ -181,6 +184,15 @@ export class PromoteNotificationTemplateChange {
       }
     );
   }
+
+  private async invalidateBlueprints(command: PromoteTypeChangeCommand) {
+    if (command.organizationId === this.blueprintOrganizationId) {
+      await this.invalidateCache.invalidateByKey({
+        key: buildGroupedBlueprintsKey(),
+      });
+    }
+  }
```

Comment:
can you clarify 🤔

Reply:
Definitely, here we are invalidating the cache, meaning that the grouped-blueprint-list record in the cache will be deleted, meaning that the next request to the API will be fetched from DB and not the cache. (after the first request it will be stored in the cache again).
My suggestion here is to manually store it again in the cache (updated data). meaning the next request will be fetched from the cache after the invalidation here.

I am not sure we need it anymore because:
1. we won't have a lot of updates.
2. this record is distributed to all the users meaning only one user will wait a bit more than expected. which feels redundant now.

Reply:
Thanks for the clarification. I think that for now, it's fine just to invalidate the cache, especially that the GET request will be happening on the main templates list page, which is used super often, then the cache will be populated right away after the update.